features/zoek-met-geslachtsnaam-geboortedatum.feature

```diff
@@ -2,522 +2,416 @@
+    | geslachtsnaam | <geslachtsnaam filter>              |
+    | geboortedatum | 1983-05-26                          |
+    | fields        | burgerservicenummer                 |
+    Dan bevat de response alleen personen met de volgende gegevens 
+    | burgerservicenummer |
+    | 999995079           |
+    | 999995083           |
+
+    Voorbeelden:
+    | geslachtsnaam filter |
+    | *oen*                |
+    | *OEN*                |
+
+  @gba
+  Abstract Scenario: Zoek met "*" wildcard als laatste karakter in voornamen
+    Als personen wordt gezocht met de volgende parameters
+    | naam          | waarde                              |
+    | type          | ZoekMetGeslachtsnaamEnGeboortedatum |
+    | geslachtsnaam | maassen                             |
+    | voornamen     | <voornamen filter>                  |
+    | geboortedatum | 1983-05-26                          |
+    | fields        | burgerservicenummer                 |
+    Dan bevat de response alleen personen met de volgende gegevens 
+    | burgerservicenummer |
+    | 999995082           |
+
+    Voorbeelden:
+    | voornamen filter |
+    | Jan*             |
+    | jan*             |
+    | JAN*             |
+
+  @fout-case
+  Abstract Scenario: Zoek met "*" wildcard als eerste karakter in voornamen
+    Als personen wordt gezocht met de volgende parameters
+    | naam          | waarde                              |
+    | type          | ZoekMetGeslachtsnaamEnGeboortedatum |
+    | geslachtsnaam | maassen                             |
+    | voornamen     | <voornamen filter>                  |
+    | geboortedatum | 1983-05-26                          |
+    | fields        | burgerservicenummer                 |
+    Dan bevat de response de volgende gegevens
+    | naam     | waarde                                                                                                      |
+    | type     | https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode?#System_Net_HttpStatusCode_BadRequest |
+    | title    | Een of meerdere parameters zijn niet correct.                                                               |
+    | status   | 400                                                                                                         |
+    | detail   | De foutieve parameter(s) zijn: geslachtsnaam.                                                               |
+    | code     | paramsValidation                                                                                            |
+    | instance | /personen                                                                                                   |
+    En bevat de response een invalidParams met de volgende gegevens
+    | code     | name      | reason                                                              |
+    | wildcard | voornamen | Waarde voldoet niet aan patroon ^[a-zA-Z0-9À-ž \.\-\']{1,199}\*{1}$ |
```

Comment:
als ik de reason lees is het een pattern fout. dan had ik eigenlijk ook als code "pattern" verwacht

Reply:
copy/paste actie van de foutmelding uit de api-lab implementatie. Is gecorrigeerd